docs/framework.md

```diff
@@ -76,14 +76,79 @@ Pass the resulting `$commandClasses` to the `Runner()` constructor as shown abov
 
 It is also possible to completely replace the Robo application with your own.  To do this, set up your project as described in the sections above, but replace the Robo runner with your own main event loop.
 
-Create the Robo dependency injection container:
-```
+Add the following to your startup file:
+```php
+<?php
 use League\Container\Container;
 
 $input = new \Symfony\Component\Console\Input\ArgvInput($argv);
 $output = new \Symfony\Component\Console\Output\ConsoleOutput();
 $conf = new \Robo\Config(); \\ or use your own subclass
-$app = new \My\Application();
+$app = new \MyApplication($config, $input, $output);
 $container = \Robo\Robo::createDefaultContainer($input, $output, $app, $conf);
```

Comment:
Did you mean to delete this line? There's a createDefaultContainer below as well.